lib/components/app/session-timeout.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
Is there something blocking typescripting this component?

Reply:
Added types in https://github.com/opentripplanner/otp-react-redux/pull/574/commits/292181c4439ad16d50a1e1860fd7504c3b1f2721.